helpers/lib/cdnify.js

```diff
@@ -26,28 +24,38 @@ module.exports = globals => {
             return '';
         }
 
-        if (/^(?:https?:)?\/\//.test(path)) {
+        if (path.startsWith('http') || path.startsWith('//')) {
             return path;
         }
-
+        //this block is to strip leading . and / of path
         if (['.', '/'].includes(path[0])) {
-            path = path.replace(/^[\.\/]+/, '');
+            for (let i = 0; i < path.length; i++) {
+                if (path[i] !== '.' && path[i] !== '/') {
+                    path = path.slice(i);
+                    break;
+                }
+            }
         }
 
-        if (protocolMatch.test(path)) {
-            var match = path.match(protocolMatch);
-            path = path.slice(match[0].length, path.length);
+        let colonIndex = path.indexOf(':');
+
+        if (colonIndex !== -1) {
+            while (path[colonIndex + 1] === ':') {
+                colonIndex++;
+            }
+            const match = path.slice(0, colonIndex + 1);
```

Comment:
I would name this `protocol` for clarity. We are no longer using a regex which returns a match array, so "match" isn't very clear.

Reply:
done.